go/history/history_test.go

```diff
@@ -97,9 +79,7 @@ func TestIsEquivalent(t *testing.T) {
 	q := New(4)
 	q.Add(duplic(0))
 	q.Add(duplic(0))
-	if got, want := len(q.Records()), 1; got != want {
-		t.Errorf("len(q.Records())=%v, want %v", got, want)
-	}
+	assert.EqualValues(t, 1, len(q.Records()))
```

Comment:
it's better to use `Len` but it's also fine as it is.
```suggestion
	assert.Len(t, q.Records(), 1)
```